mne_bids/report.py

```diff
@@ -201,10 +201,13 @@ def _summarize_participants_tsv(root, verbose=True):
     # summarize sex count statistics
     keys = ['M', 'F', 'n/a']
     p_sex = participants_tsv.get('sex')
+    # phrasing works for both sex and gender
+    p_gender = participants_tsv.get('gender')
```

Comment:
why did you add this "gender" column?

Reply:
That's how the openneuro dataset in this PR https://github.com/mne-tools/mne-bids/pull/753 codes sex/gender

Reply:
I think it's not the only participants tsv that might have gender instead of sex

Reply:
ok, fine with me :+1: